### PR TITLE
feat(smart-contracts): add polygon support for contract testing

### PR DESCRIPTION
**Pull request by clemsos**
# Description

Can be run with `RUN_FORK=137`

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: smart-contracts/test/Unlock/uniswapValue.mainnet.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ contract('Unlock / uniswapValue', () => {
     unlock = await ethers.getContractAt('Unlock', unlockAddress)
 
     // deploy oracle
-    oracle = await deployUniswapV3Oracle()
+    oracle = await deployUniswapOracleV3()
```
Comment (julien51) [inline smart-contracts/test/Unlock/uniswapValue.mainnet.js:47]: Why the change?

**Review by julien51 — APPROVED** [review]

File: smart-contracts/test/Unlock/uniswapValue.mainnet.js (hunk 1 above)
Comment (clemsos) [reply smart-contracts/test/Unlock/uniswapValue.mainnet.js]: was a typo in the name I discovered when running the test but will fix in the lib to keep `UniswapV3` in the name.
